test/e2e/kubectl/kubectl.go

```diff
@@ -544,7 +544,7 @@ var _ = SIGDescribe("Kubectl client", func() {
 				if !ok {
 					framework.Failf("Got unexpected error type, expected uexec.ExitError, got %T: %v", err, err)
 				}
-				framework.ExpectEqual(ee.ExitStatus(), 42)
+				gomega.Expect(ee.ExitStatus()).To(gomega.Equal(42))
```

Comment:
Thanks for the PR.

I think if you're going to do this cleanup, let's replace all deprecated framework function calls within these two files (kubectl.go and logs.go).

This includes:
* framework.ExpectEqual
* framework.ExpectNotEqual
* framework.ExpectError

(There are other deprecated functions used in these files, which are not framework-related, but I think it is fine to leave those as-is in this PR)